.github/workflows/main.yml

```diff
@@ -3,58 +3,58 @@ name: "CI Multiplatform Build"
 on:
   push:
     paths-ignore:
-      - 'docs/**'
+      - "docs/**"
     branches:
-      - master
+      - v5-macos-ci
     tags:
-      - '[45].[0-9]+.[0-9]+*'
+      - "[45].[0-9]+.[0-9]+*"
   pull_request:
     paths-ignore:
-      - 'docs/**'
-      - '**.md'
+      - "docs/**"
+      - "**.md"
     branches:
       - master
 
 jobs:
   windows:
-    name: 'Windows 32/64-bit'
+    name: "Windows 32/64-bit"
     runs-on: [windows-latest]
     if: contains(github.event.head_commit.message, '[skip ci]') != true
     env:
-      QT_CACHE_VERSION: '2' # Change whenever updating OBS dependencies URL, in order to force a cache reset
-      QT_VERSION: '5.15.2'
-      WINDOWS_DEPS_CACHE_VERSION: '1' # Change whenever updating Qt dependency URL, in order to force a cache reset
-      WINDOWS_DEPS_VERSION: '2019'
+      QT_CACHE_VERSION: "2" # Change whenever updating OBS dependencies URL, in order to force a cache reset
+      QT_VERSION: "5.15.2"
```

Comment:
Is there a reason why all of the `'` have been replaced with `"`?

Reply:
That would be my backed formatter 😅